stingray/simulator/tests/test_simulator.py

```diff
@@ -57,6 +58,13 @@ def test_simulate_channel(self):
         self.simulator.simulate_channel('3.5-4.5', 'lorenzian', [1, 2, 3, 4])
         self.simulator.delete_channel('3.5-4.5')
 
+    def test_simulate_channel_odd(self):
+        """
+        Simulate an energy channel.
+        """
+        self.simulator_odd.simulate_channel('3.5-4.5', 'lorenzian', [1, 2, 3, 4])
```

Comment:
Hm, that's annoying! The simulator takes a string argument that has a typo! Should be lorentzian with a t before the z...